crates/conjure_core/src/ast/expressions.rs

```diff
@@ -103,6 +110,10 @@ pub enum Expression {
     #[compatible(Minion)]
     DivEq(Metadata, Atom, Atom, Atom),
 
+    /// `a % b = c`
+    #[compatible(Minion)]
+    ModEq(Metadata, Atom, Atom, Atom),
```

Comment:
this didn't occur to me last time with DivEq, but why not use the same name as in the minion binding? i.e. ModuloUndefZero? same with DivUndefZero. is it to do with wanting to avoid namespaces perhaps? a different name introduces additional cognitive load, one has to spend time to understand it's actually the same semantics (except for the types of course).